cranelift-codegen/src/lib.rs

```diff
@@ -48,6 +48,8 @@ extern crate alloc as std;
 #[cfg(feature = "std")]
 #[macro_use]
 extern crate std;
+#[macro_use]
+extern crate log;
```

Comment:
I'm fine either way; @sunfishcode was helping me debug and this is what we did... I'll change it to the more explicit `use log::debug` unless someone tells me otherwise.